site/docs/expressions/field_references.md

```diff
@@ -11,6 +11,59 @@ In Substrait, all fields are dealt with on a positional basis. Field names are o
 | Map KeyExpression         | A wildcard string that is matched against a simplified form of regular expressions. Requires the key type of the map to be a character type. [Format detail needed, intention to include basic regex concepts such as greedy/non-greedy.] | map                | List of map value type     |
 | Masked Complex Expression | An expression that provides a mask over a schema declaring which portions of the schema should be presented. This allows a user to select a portion of a complex object but mask certain subsections of that same object. | any                | any                        |
 
+
+```
+Schema:
+a: struct
+  b: list
+    c: map string
+      x: i32
+
+SQL expression:
+a.b[2].c['my_map_key'].x
+```
+
+```
```

Comment:
I couldn't find a way to set this block as protobuf text. Maybe we could show the example as JSON.

https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml#L5370

Reply:
JSON would be good I think.  In many places we are hoping to add examples using the text serialization but I don't think that would be helpful here since we want to demonstrate a nuance of how to specify this in protobuf.

Reply:
The downside of JSON is the lack of comments. I kept Protobuf text for now.